### PR TITLE
feature: slide text animation for homepage hero

### PR DESCRIPTION
**Pull request by bhongy**
https://user-images.githubusercontent.com/3942264/218726173-ad8f4850-00f1-40f4-b8ed-4c8c0acdd646.mov

### CODE REVIEW

**Comment by bhongy** [comment]
@iwasrobbed can you check the line-height? I think with the animation; it's a bit close to the second line.

File: src/styles/globals.css [hunk 1]
```diff
@@ -12,3 +12,35 @@
...
+.text-animation-exit {
+  opacity: 1;
+  transform: translate3d(0, 0%, 0);
+}
+
+.text-animation-exit-active {
+  opacity: 0;
+  transform: translate3d(0, -50%, 0);
+  transition: opacity 360ms ease-in-out, transform 360ms ease-in-out;
+}
```
Comment (bhongy) [inline src/styles/globals.css:46]: Just for now. We'll figure out where to put the tailwind way in the future.

**Comment by iwasrobbed** [comment]
See warning [here](https://usevenice.slack.com/archives/C04P51BSEDP/p1676394686392429) ; can punt on this for now unless it's a super quick fix

**Comment by bhongy** [comment]
Punt for now; @iwasrobbed saw a warning in console (can you add screenshot)? I couldn't repro.

**Pull request closed without merging**